tasks/main.yml

```diff
@@ -1,25 +1,32 @@
 # SPDX-License-Identifier: MIT
 ---
+
 - name: Set platform/version specific variables
   include_tasks: tasks/set_vars.yml
 
-# Examples of some tasks:
 - name: Ensure required packages are installed
   package:
-    name: "{{ __template_packages }}"
+    name: "{{ __keylime_server_packages }}"
     state: present
 
+- name: Generate keylime legacy configuration
+  template:
+    src: keylime.conf.j2
+    dest: /etc/keylime.conf
+    backup: false
+    mode: "0600"
+    owner: "{{ __keylime_server_user }}"
+    group: "{{ __keylime_server_group }}"
+  notify: Restart services
+  when: __keylime_server_legacy_config | bool
+
+- name: Generate keylime configuration
+  include_tasks: keylime-config.yml
+  when: not (__keylime_server_legacy_config | bool)
```

Comment:
if `__keylime_server_legacy_config` is a boolean valued variable, you don't need to use the `| bool` here since the `not` makes this "not a bare variable" - so `when: not __keylime_server_legacy_config`

Reply:
Done.